client/src/components/AddMovieForm/AddMovieForm.spec.js

```diff
@@ -0,0 +1,31 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import AddMovieForm from './AddMovieForm'
+
+describe('AddMovieForm', () => {
+  it('renders', () => {
+    render(<AddMovieForm />)
+
+    const h2element = screen.getByRole('heading', { level: 2 })
+    expect(h2element).toBeInTheDocument()
+
+    const labelItem = screen.getByLabelText('TMDB Id')
+    expect(labelItem).toBeInTheDocument()
+
+    const buttonElement = screen.getByRole('button')
+    expect(buttonElement).toBeInTheDocument()
+  })
+
+  // Does not work yet:
+  // Cannot find module '@testing-library/dom'
+  // from 'node_modules/@testing-library/user-event/dist/click.js'
```

Comment:
Urgh, I hate when that happens! I think you can install this in your client folder and then it should work!
```
cd client && npm install @testing-library/dom
```
I had this issue once as well, but couldn't find out why it was missing 🤔. This solved it for me.